backend/src/db/entities/lastwill.entity.ts

```diff
@@ -272,6 +274,7 @@ class PersonBase {
     type: String,
   })
   @IsOptional()
+  @ValidateIf((e) => e.birthDate !== '')
```

Comment:
But does it validate then if its null or undefined? Or doesnt matter coz ISO can also be undefined?

Reply:
Undefined values are removed anyways by serialisation, no?